packages/core/src/extension/extension-post-event.ts

```diff
@@ -0,0 +1,26 @@
+export type ExtensionCommand = 'RELOAD_CHECKOUT' | 'SHOW_LOADING_INDICATOR';
+
+export type HostCommand = 'HOST_COMMAND';
```

Comment:
What's a `HostCommand`? If it's intended to be issued by the host to the extension rather than the extension to the host, it'd be good to be explicit about what action you want to perform. Also, to make it clearer who's the recipient of a command, I'd split the `ExtensionPostEvent` into two, one is exclusively for the extension -> host, the other for the host -> extension.

Reply:
`HostCommand` was created for client-side use.
Since we are spliting them now, I have removed it and renamed `ExtensionPostEvent` as `ExtensionOriginEvent`.